src/css/components/_progress.scss

```diff
@@ -32,6 +32,10 @@
   @include flex(auto);
   @include transition(all 0.2s);
   height: 0.3em;
+
+  * {
+    pointer-events: none;
```

Comment:
This was because when you clicked on elements inside `progress-holder` such as chunks of `vjs-load-progress` it will throw a different position, but with the latest changes using only offsetPositions, this is no longer needed.